code/src/components/MainGame.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { useSelector, useDispatch} from "react-redux";
+import { fetchGameSteps } from '../reducers/game';
+
+import { useNavigate } from 'react-router-dom';
+import { MainGameStyle, DirectionSection, WinSection, DirectionButton, InfoSection } from "./Style"
+
+const MainGame = () => {
+    const gameObject = useSelector((store) => store.game.gameObject)
+    const username = useSelector((store) => store.game.username)
+    const loading = useSelector((state) => state.ui.loading)
+    const navigate = useNavigate()
+
+    const dispatch = useDispatch()
+
+    const onRestartClick = () => {
+        navigate('/')
+    }
+
+    return (         
+        <div>
+            {!loading && (
+            <MainGameStyle>
+
+                <InfoSection className="nes-container with-title is-centered">
+                <p className="title" tabIndex='0'>User: {username} </p>
+                <p tabIndex='0'>{gameObject.description}</p>
```

Comment:
Keyboard accessibility = 🥳 Great!

Reply:
Like Camilla said, nice with the keyborad access :)